src/platform/forms-system/src/js/helpers.js

```diff
@@ -28,8 +28,8 @@ export function isActivePage(page, data) {
   return page.depends === undefined || matches(page.depends)(data);
```

Comment:
I think this line and the line above (line 25) needs to include the additional parameter

Reply:
Yeah there are quite a few places that it needs to be added to. This is really just an MVP to get across the idea. I'd like to make sure that this is viable before putting more work into it

Reply:
But if this is something that platform would like to see done, I will definitely go through the code and make sure it's added in all of the places it needs to be